include/robot_localization/ros_filter.h

```diff
@@ -549,8 +558,35 @@ template<class T> class RosFilter
     //! @brief tf frame name that is the parent frame of the transform that this node will calculate and broadcast.
     //!
     std::string worldFrameId_;
-};
 
+    //! @brief Publisher to publish the filtered pose
+    //!
+    ros::Publisher positionPub_;
```

Comment:
pedantic: this actually publishes `Odometry` (which contains twist and pose, where pose is composed of position and orientation). To avoid any confusion, I'd renamed this to `odometryPub_` or `filteredPub_`